pkg/server/share_manager.go

```diff
@@ -265,6 +264,22 @@ func (m *ShareManager) recoverReadOnlyVolume() error {
 	return nil
 }
 
+func (m *ShareManager) GetVolume() volume.Volume {
+	return m.volume
+}
+
+func (m *ShareManager) SetShareExported(val bool) {
+	m.shareExported = val
```

Comment:
Just remember this is not thread safe but it should be fine. I assume mount/unmount should not be executed concurrently.

Reply:
Yes, but let me add a lock for the potential concern.